.gitignore

```diff
@@ -137,3 +137,4 @@ resources/
 
 # Snakemake
 .snakemake/
+./openproblems/tasks/dimensional_reduction/examples/citeseq_rmse/citeseq_processed_rmse_calculated.h5ad
```

Comment:
Would it make sense to just gitignore *.h5ad? We never want data files in this repo, I think.

Reply:
Agreed, changed to that - great suggestion.